### ISSUE TITLE
I only get a random image being render everywhere with random sizes

### ISSUE THREAD

**Issue by pmgou** [juangallostra/augmented-reality#12]

**Comment by pmgou** [comment]
changed https://github.com/juangallostra/augmented-reality/blob/master/src/ar_main.py#L20 to 200 and helped!

**Comment by pmgou** [comment]
I find the tracking not very smooth, don't you apply any filter so that its a bit smoother the tracking of movement?

**Comment by juangallostra** [comment]
@pmgou yep, this is a known issue. Currently there is no tracking being applied, the reference surface is detected each frame without taking into account information from the previous frame. 

Tracking is something that is on my to-do list though. I hope to find some time to work on it.

**Comment by WiiliamC** [comment]
I get the same issue. Did you change the camera_parameters in https://github.com/juangallostra/augmented-reality/blob/19a49c47982df9009c7f6ec06a0eb3613050562b/src/ar_main.py#L29? Is there any necessity to do this when I use my own model and video?

**Comment by juangallostra** [comment]
@WiiliamC changing the camera parameters might help. However, I would expect it to work with the default parameters included in the code. I suggest to check each step of the process individually to try and find were the problem is.

The first thing I would check is how good the matches between the reference surface and a video frame are. If the obtained matches are not accurate that might be the cause of the issue. If they are, we can jump to the next step of the process.
